armi/physics/fuelCycle/tests/test_assemblyRotationAlgorithms.py

```diff
@@ -19,36 +19,315 @@
 These algorithms are defined in assemblyRotationAlgorithms.py, but they are used in:
 ``FuelHandler.outage()``.
 """
+import copy
+import enum
+import math
+import typing
+from unittest import mock, TestCase
+
+import numpy as np
+
+from armi.reactor.assemblies import HexAssembly
+from armi.reactor.blocks import HexBlock
 from armi.physics.fuelCycle import assemblyRotationAlgorithms as rotAlgos
+from armi.physics.fuelCycle.hexAssemblyFuelMgmtUtils import (
+    getOptimalAssemblyOrientation,
+)
 from armi.physics.fuelCycle import fuelHandlers
 from armi.physics.fuelCycle.settings import CONF_ASSEM_ROTATION_STATIONARY
 from armi.physics.fuelCycle.tests.test_fuelHandlers import addSomeDetailAssemblies
-from armi.physics.fuelCycle.tests.test_fuelHandlers import FuelHandlerTestHelper
 from armi.reactor.flags import Flags
+from armi.reactor.tests import test_reactors
 
 
-class TestFuelHandlerMgmtTools(FuelHandlerTestHelper):
-    def test_buReducingAssemblyRotation(self):
-        fh = fuelHandlers.FuelHandler(self.o)
-        hist = self.o.getInterface("history")
-        newSettings = {CONF_ASSEM_ROTATION_STATIONARY: True}
+class MockFuelHandler(fuelHandlers.FuelHandler):
+    """Implements the entire interface but with empty methods."""
+
+    def chooseSwaps(self, *args, **kwargs):
+        pass
+
+
+class PinLocations(enum.IntEnum):
```

Comment:
In test classes, I like to make sure all "test class" names have words like "Mock" or "Test" or "Faux" or something in them. That way, people don't import them and use them in non-test code.

(I have caught people doing that in the past.)

Reply:
Thoughts on `_PinLocations`? It's not a `unittest.TestCase` "test class" but I see your concern.

> In test classes, I like

Thoughts on adding that preference to the dev guide?

Reply:
It doesn't have to be a `TestCase` subclass for people to import it in their code.  ...Sadly.

That's why I put `Mock` in front of things like this.

But, sure, if you like, you can make it a private function too.  That serves the same purpose.